src/analyzer/protocol/irc/IRC.cc

```diff
@@ -127,6 +142,10 @@ void IRC_Analyzer::DeliverStream(int length, const u_char* line, bool orig) {
     // Extract parameters.
     string params = myline;
 
+    if ( ! AnalyzerConfirmed() && orig && IsValidClientCommand(command) ) {
```

Comment:
There's logic [further down](https://github.com/zeek/zeek/blob/master/src/analyzer/protocol/irc/IRC.cc#L493-L502) in this file that looks at the command length to determine validity. Is that obsolete now that we have an explicit command list?

Reply:
Not really - the list is only used for confirmation to be minimally invasive.